package cz.opendatalab.egidio.backend.business.exceptions.not_found

class UserNotFoundException : EntityNotFoundException("User")